src/pages/carbon-offsetting.jsx

```diff
@@ -0,0 +1,564 @@
+            <a href="http://news.mit.edu/2010/explained-radforce-0309">
+              Emissions at altitude have a compounding effect
+            </a>{' '}
+            which make flying{' '}
+            <a href="https://www.bbc.com/news/science-environment-49349566">
+              the worst way to get around in most scenarios
+            </a>{' '}
+            (other than, for example, a short jump over a mountain range versus
+            a very long drive around the mountain range).
+          </p>
+
+          <p>
+            Nobody likes to hear this but the solution genuinely is to stop
+            flying. 2% of the worlds emissions come from air travel, but only
+            10% of people in the world fly. 2% is the same amount as{' '}
+            <a href="https://informationisbeautiful.net/visualizations/how-to-reduce-the-worlds-carbon-footprint-by-2050/">
+              processing the entire worlds sewage
+            </a>
+            , and 100% of the world are involved in that. The price is
+            unrealistically low due to taxpayers subsidizing the costs, and why
+            should we all pay for 15% of people to jet about on fancy holidays,
+            putting all of our health and existence at risk.
+          </p>
+
+          <p>
+            Offsetting cannot make flying sustainable, and airlines suggesting
+            so are just trying fix their reputation with the increasingly
+            climate-conscious public, and are doing so by passing the cost on to
+            you. They might pocket your money by doing literally nothing, or
+            maybe they'll do "not enough" with it, but offsetting definitely is
+            not the answer here. We need an aviation carbon tax or at the very
+            least a "frequent flier tax", which will raise flying to a realistic
+            price and in turn provide funding for electric trains.{' '}
+          </p>
+
+          <p>
+            Flying could be 25% of your carbon footprint for the year, so it's
+            probably the lowest hanging fruit for personal change. Climate
+            activists like George Manibot reduce their flying to once every
+            three years, but many Americans fly more than once a month, and no
+            amount of money can soak up that rate of emission.
+          </p>
+
+          <p>
+            Airlines aren't the only polluting industry excited to market
+            offsetting as a solution to their bad press, petrol stations are
+            getting in on the greenwashing too.
+          </p>
+
+          <img
+            src={imgShell}
+            alt="Shell poster promoting an app where you can carbon offset your driving"
+            style={{ width: '100%' }}
+          />
+
+          <p>
+            Just because the polluting industries support offsetting, doesn't
+            make offsetting a bad idea. They're generally pushing a weak
```

Comment:
Replce with 'Just because the polluting industries **claim** to support offsetting, **it** doesn't **mean** offsetting **is** a bad idea.  They're generally pushing a **weak, unprovable, and cheap** version of it to **greenwash** their marketing more than anything else.